admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -703,13 +703,20 @@ Those mappings are done in the database table ``ldap_user_mapping`` and
 if something else is specified in *User Home Folder Naming Rule*), which
 contains files and meta data.
 
-As of Nextcloud 5 the internal user name and a visible display name are separated.
+The internal user name and a visible display name are separated.
 This is not the case for group names, yet, i.e. a group name cannot be altered.
 
 That means that your LDAP configuration should be good and ready before putting
 it into production. The mapping tables are filled early, but as long as you are
 testing, you can empty the tables any time. Do not do this in production.
 
+The attributes of all users are fetched on demand and then stored inside the
+Nextcloud database to not overload the LDAP server with requests. They are
```

Comment:
It is not about request against LDAP server (searches and existence checks are done anyway unless caches), but performance on our side. Checking and storing data on sequentially on several places for several records has an impact on UX, therefore it is brought to the background. For login's it is less of a problem, as it is just one user being updated.

Reply:
Refined the wording.